src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        Color randomColor = colors[new Random().nextInt(colors.length)];
```

Comment:
`Random` should be `private final` class-level field
```suggestion
        Color randomColor = colors[random.nextInt(colors.length)];
```